pkg/ebpf/common.go

```diff
@@ -19,8 +19,6 @@ var requiredKernelFuncs = []string{
 	"bpf_map_lookup_elem",
 	"bpf_map_update_elem",
 	"bpf_map_delete_elem",
-	// kprobes (4.1)
-	"bpf_probe_read",
```

Comment:
Can we add a comment like "bpf_probe_read  intentionally omitted since it was renamed in kernel 5.5"

Reply:
Sure. Done